### ISSUE TITLE
Catch2 documentation says "This limitation is highly unlikely to be lifted before Catch 2 is released"

### ISSUE THREAD

**Issue by RedDwarf69** [catchorg/Catch2#1302]
I'm just reading about Catch for first time. Finding "This limitation is highly unlikely to be lifted before Catch 2 is released" in https://github.com/catchorg/Catch2/blob/master/docs/limitations.md is confusing.

**Comment by onqtam** [comment]
The idea is that using Catch asserts in threads before they are joined is not supported in Catch 1, and will be supported at some point in Catch 2 but that isn't the case yet. You still cannot use asserts from multiple threads.

**Comment by horenmar** [comment]
Pretty much the above: There was a slight possibility that someone would be crazy enough to implement it in C++98, or add extra compile time configuration branches for it. At some point it should become part of Catch2, as C++11 makes it much easier, but it is not in yet.

How did you read it? Would it help if it said something along the lines of "not supported in Catch v1.x, not going to be supported in Catch v1.x, but there are plans for support in Catch v2.x"?

**Comment by RedDwarf69** [comment]
Yes, that would be more clear.

"This limitation is highly unlikely to be lifted before Catch 2 is released"... Well it's not "highly unlikely", it's literally impossible. The probability is exactly zero. Catch 2 has already been released. So I reached the conclusion that the text was out of date, the information was not reliable and so it was equally possible that Catch 2 supported or not supported thread safe assertions.
